### ISSUE TITLE
Ecoute -  real-time transcription tool

### ISSUE THREAD

**Issue by UjwalKandi** [UjwalKandi/Issue-Bookmarks#402]
**TL;DR** - Ecoute is a live transcription tool that provides real-time transcripts for both the user's microphone input (You) and the user's speaker output (Speaker) in a textbox. It also generates a suggested response using OpenAI's GPT-3.5 for the user to say based on the live transcription of the conversation.

Ecoute is designed to help users in their conversations by providing live transcriptions and generating contextually relevant responses. By leveraging the power of OpenAI's GPT-3.5, Ecoute aims to make communication more efficient and enjoyable.

**Website:**
https://github.com/SevaSk/ecoute
